cucumber-messages/javascript/.gitignore

```diff
@@ -5,7 +5,6 @@ coverage/
 node_modules/
 yarn.lock
 npm-debug.log
-package-lock.json
```

Comment:
Not sure if this was intentional originally, but meant volatile builds were possible.